documentation/topics/advanced/multitenancy.md

```diff
@@ -7,7 +7,7 @@ In Ash, there are two primary strategies for implementing multitenancy. The firs
 ## Attribute Multitenancy
 
 ```elixir
-defmodule MyApp.Users do
+defmodule MyApp.Accounts.User do
```

Comment:
I can't propose it as an edit here, but let's change the organization reference below to `MyApp.Accounts.Organization`. Just more idiomatic Ash naming.